modular_skyrat/modules/vore/code/datums/vore_prefs.dm

```diff
@@ -0,0 +1,584 @@
+//original code by: Seris02
+
+/datum/vore_prefs
```

Comment:
Move this entire setup to tgui preferences. I will assist with doing so if needed.

We already had one disasterous preferences and save file hiccup from weird extra menus, let's not have a second.